Tests/Pcap++Test/Tests/KniTests.cpp

```diff
@@ -239,37 +256,40 @@ PTF_TEST_CASE(TestKniDevice)
 		}
 	}
 	{
-		pcpp::KniDevice::KniLinkState ls = device->updateLinkState(isLinkUp ? pcpp::KniDevice::LINK_DOWN : pcpp::KniDevice::LINK_UP);
+		pcpp::KniDevice::KniLinkState ls =
+		    device->updateLinkState(isLinkUp ? pcpp::KniDevice::LINK_DOWN : pcpp::KniDevice::LINK_UP);
 		switch (ls)
 		{
-			case pcpp::KniDevice::LINK_NOT_SUPPORTED:
-			{
-				PTF_PRINT_VERBOSE("KNI updateLinkState not supported");
-			} break;
-			case pcpp::KniDevice::LINK_ERROR:
-			{
-				PTF_PRINT_VERBOSE("KNI updateLinkState have failed with LINK_ERROR");
-			} break;
-			case pcpp::KniDevice::LINK_DOWN:
-			{	// If previous known state was UP -> yield an error
-				PTF_ASSERT_FALSE(isLinkUp);
-			} break;
-			case pcpp::KniDevice::LINK_UP:
-			{	// If previous known state was DOWN -> yield an error
-				PTF_ASSERT_TRUE(isLinkUp);
-			} break;
+		case pcpp::KniDevice::LINK_NOT_SUPPORTED:
+		{
+			PTF_PRINT_VERBOSE("KNI updateLinkState not supported");
+		}
+		break;
+		case pcpp::KniDevice::LINK_ERROR:
+		{
+			PTF_PRINT_VERBOSE("KNI updateLinkState have failed with LINK_ERROR");
+		}
+		break;
+		case pcpp::KniDevice::LINK_DOWN:
+		{  // If previous known state was UP -> yield an error
+			PTF_ASSERT_FALSE(isLinkUp);
+		}
+		break;
+		case pcpp::KniDevice::LINK_UP:
+		{  // If previous known state was DOWN -> yield an error
+			PTF_ASSERT_TRUE(isLinkUp);
+		}
+		break;
```

Comment:
Maybe we can move the `break` to inside the curly brackets?

Reply:
fixed all cases.